src/main/java/javaserver/ResponseBuilders/Responder.java

```diff
@@ -3,9 +3,8 @@
 import javaserver.Request;
 import javaserver.Response;
 
-public interface ResponseBuilder {
+public interface Responder {
 
 	public Response getResponse(Request request);
-	public void setResponseData(Request request);
 	public String getStatusLine(Request request);
```

Comment:
I noticed that all of your Responders have the private method `requestIsSupported`. Since you're using Java 8, you could add it as a default method inside this interface. Something like this:

```java
default public boolean requestIsSupported(String[] supportedMethods, String method) {
  return Arrays.asList(supportedMethods).contains(method);
}
```